site/content/en/docs/drivers/docker.md

```diff
@@ -19,19 +19,13 @@ The Docker driver allows you to install Kubernetes into an existing Docker insta
 
 ## Known Issues
 
-- On macOS, containers might get hung and require a restart of Docker for Desktop. See [docker/for-mac#1835](https://github.com/docker/for-mac/issues/1835)
```

Comment:
why did u delete this one, I think you wanted to update the line bellow.

Reply:
I restore that file